app/jobs/occupation_update_job.rb

```diff
@@ -0,0 +1,16 @@
+class OccupationUpdateJob < ApplicationJob
+  queue_as :occupation
+
+  def perform
+    Sidekiq.logger.debug('Starting occupation update')
+    ApaFetcher.occupation
+    closing_time = Time.zone.parse('21:00')
+    next_update = if Time.current.now >= closing_time
+                    Time.zone.parse('8:00') + 1.day
```

Comment:
@simaob these values could be constants (and maybe start earlier than 8am... 6am would be better, I guess)